src/components/clients/ClientForm.js

```diff
@@ -18,6 +27,28 @@ class ClientForm extends React.Component {
     });
   }
 
+  onRedirectUrlChange(e) {
+    const focus = this.state[e.target.name + 'Focus'];
+    let urls = Object.assign([], this.state[e.target.name]);
+    urls[focus] = e.target.value;
+    this.setState({
+      [e.target.name]: urls
+    });
+  }
+
+  onClick(e) {
+    e.preventDefault();
+    this.setState({
+      [e.target.name]: this.state[e.target.name] + 1
+    });
+  }
+
+  onFocus(e) {
+    this.setState({
+      [e.target.name + 'Focus']: parseInt(e.target.id, 10)
+    });
+  }
```

Comment:
I don't understand why this logic with `onFocus` is needed. Isn't it possible to just use `e.target.id` in `onRedirectUrlChange` ?